test/robot/replay/client.go

```diff
@@ -59,6 +59,11 @@ func Run(ctx context.Context, store *stash.Client, manager Manager, tempDir file
 func (c *client) onDeviceAdded(ctx context.Context, host *device.Instance, target bind.Device) {
 	replayOnTarget := func(ctx context.Context, t *Task) error {
 		job.LockDevice(ctx, target)
+		defer func() {
+			// TEMP: kill gapid.apk manually for now as subsequent reports/replays may freeze the app.
```

Comment:
Add `HACK:`, please link to a bug that tracks the issue and gives as much information as possible.